build.cmd

```diff
@@ -1,41 +1,4 @@
-@if not defined _echo @echo off
-setlocal
-
-if /I [%1] == [-?] goto Usage
-
-if [%1] == [] goto Build
-
-set _rootDirectory=%CD%\
-if EXIST %_rootDirectory%%1 goto :BuildDirectory
-set _rootDirectory=%~dp0src\
-if EXIST %_rootDirectory%%1 goto :BuildDirectory
-
-:Build
-call %~dp0build-native.cmd %*
-if NOT [%ERRORLEVEL%]==[0] exit /b 1
-call %~dp0build-managed.cmd -BuildPackages=true %*
+@echo off
```

Comment:
These scripts are basically a single line and then call powershell scripts so that pattern isn't really going to help unless we can plumb it through.